### DISCUSSION TITLE
backup mysql coordinate garbled

### DISCUSSION THREAD

**Discussion by lengjunbin-trec** [spatie/laravel-backup#1690] [Q&A] [answered]
when I try  to bacp up mysql data for coordinate.the data is garbled.
source data  : `POINT (130.4503597 33.6254727)`
backed up data ： æ\0\0\0\0\0÷Ý¾XiN`@:…K}Ð@@

 "spatie/laravel-backup": "^6.16",
 "laravel/framework": "^7.0",
  "php": "^7.4",

**Comment by freekmurze** [comment, marked as answer]
I'd accept a PR that fixes this.

**Reply by lengjunbin-trec** [reply]
@freekmurze I have fixed this issue in my local.but I don't have permission to push this fix into github.

**Reply by parallels999** [reply]
> I don't have permission to push this fix into github.

you have to do a PR(pull request)

**Reply by lengjunbin-trec** [reply]
https://github.com/spatie/laravel-backup/pull/1698
PR has been created.

**Comment by parallels999** [comment]
>"spatie/laravel-backup": "^6.16",
>"laravel/framework": "^7.0",

Really old versions, current is `laravel-backup:^8.1` and laravel/framework 9.x or 10.x
